devtools/install_arc.sh

```diff
@@ -1,21 +1,50 @@
 # Properly configure the shell to use 'conda activate'.
 CONDA_BASE=$(conda info --base)
 source $CONDA_BASE/etc/profile.d/conda.sh
+###Check if Mamba is installed####
+echo "Determining whether Mamba is installed..."
+if [ $(which mamba) ] >/dev/null 2>&1; then
+    echo "Mamba found!"
+    COMMAND_PKG=mamba
+else
+    echo "Reverting to Conda..."
+    COMMAND_PKG=conda
+fi
+
+
+
+
 
 # Temporarily change directory to $HOME to install software
 pushd .
 
 cd ..
-git clone https://github.com/ReactionMechanismGenerator/RMG-database
-git clone https://github.com/ReactionMechanismGenerator/RMG-Py
-cd RMG-Py
+if [ -d "./RMG-database" ]; then
+    cd RMG-database
+    git pull https://github.com/ReactionMechanismGenerator/RMG-database
+    cd ..
+else
+    git clone https://github.com/ReactionMechanismGenerator/RMG-database
+fi;
+
+if [ -d "./RMG-Py" ]; then
+    cd RMG-Py
+    git pull https://github.com/ReactionMechanismGenerator/RMG-Py
+else
+    git clone https://github.com/ReactionMechanismGenerator/RMG-Py
+    cd RMG-Py
+fi;
+
+
 export PYTHONPATH=$PYTHONPATH:$(pwd)
 echo "export PYTHONPATH=$PYTHONPATH:$(pwd)" >> ~/.bashrc
-mamba env update -n rmg_env -f environment.yml
+
+
+$COMMAND_PKG env update -n rmg_env -f environment.yml
 conda activate rmg_env
 make
 python -c "import julia; julia.install(); import diffeqpy; diffeqpy.install()"
-julia -e 'using Pkg; Pkg.add(PackageSpec(name="ReactionMechanismSimulator",rev="main")); using ReactionMechanismSimulator;'
```

Comment:
In the [docs ](http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/anacondaDeveloper.html#installation-by-source-using-anaconda-environment-for-unix-based-systems-linux-and-mac-osx)they have `using ReactionMechanismSimulator`. Is `Pkg.build("ReactionMechanismSimulator")` better?

Reply:
In this instance it is kinda the same thing. in this instance. `using ReactionMechanismSimulator` is essntially Julia's way of importing a package like one would do in python - `import ReactionMechanismSimulator`. It isn't doing anything unless you then use the function in a script at that moment.

Also, after this update, I want to create a sys.so image of the julia packages to improve run time. But that's for the next PR